packages/lib/src/components/Card/components/CardInput/components/AvailableBrands/AvailableBrands.scss

```diff
@@ -6,23 +6,28 @@
   flex-shrink: 1;
   flex-wrap: wrap;
   gap: 4px;
+  height: 16px;
+  overflow: hidden;
   margin-top: -8px;
   margin-bottom: 16px;
+  will-change: height;
```

Comment:
First time seeing this property. Is this doing something in the code? MDN says that it should be used as last resort if we face performance issues.

Reply:
I use this on animated properties that will shift the layout, it tells the browser it will change so it can anticipate this change. But you're right, I didn't check the MDN article in a long time and after rereading it, the `will-change` property is definitely not warranted here.